samples/havlak.cr

```diff
@@ -183,7 +183,7 @@ class UnionFindNode
       node = parent
     end
 
-    node_list.each { |iter| iter.parent = node.parent }
+    node_list.each(&.parent=(node.parent))
```

Comment:
Most of these are great, but I'm not certain I like changing the assignments.

Reply:
I agree. Assignments with parentheses look weird.

Reply:
I've force-pushed with assignments shorthands removed.